composer.json

```diff
@@ -3,12 +3,12 @@
     "description": "PHP CS Fixer config for Paysera conventions",
     "type": "library",
     "require": {
-        "php": ">=7.4",
+        "php": "^8.1",
```

Comment:
Because I'm using a new token constant T_READONLY (not available in 7.4), and new changes are made for work with PHP 8+ features.
This is the result of running the fixer on PHP 7.4.
![image](https://github.com/user-attachments/assets/c55aa03e-bead-4823-bc5d-ae4f69b65fba)

Php-cs-fixer also has some issues with PHP 8.0. This is from its documentation:
PHP CS Fixer is not able run on PHP 8.0.0 due to bug in PHP tokenizer (https://bugs.php.net/bug.php?id=80462).

That's why I made these changes.

Reply:
I see some potential workarounds available, eg. \PhpCsFixer\Fixer\ClassNotation\OrderedClassElementsFixer::getElements
```
if (\defined('T_READONLY') && $token->isGivenKind(T_READONLY)) { // @TODO: drop condition when PHP 8.1+ is required
    $element['readonly'] = true;
}
```
This should not be a problem if CS Fixer runs on PHP 7.4 and is processing PHP 7.4 compatible code, as that version of code won't have `T_READONLY` operator.

Reply:
I saw this "defined('T_READONLY')" construction in php-cs-fixer code. And I used it to fix this issue.